src/lib/libraries/costumes.json

```diff
@@ -276,6 +276,116 @@
             1
         ]
     },
+    {
+        "name": "Bear-a",
+        "md5": "5a4148d7684fc95f38c58a1672062c9e.svg",
+        "type": "costume",
+        "tags": [],
```

Comment:
@carljbowman Did you want to tag the costumes as well?

Reply:
Weird. Those had tags, maybe fixing the spacing issue will resolve this one.